src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -389,6 +390,24 @@ export const mapRequestToState = (request: BlueprintResponse): wizardState => {
   };
 };
 
+export function mapToCustomRepositories(
+  repo: ApiRepositoryImportResponseRead
+): CustomRepository[] {
+  if (!repo.uuid) return [];
+  return [
+    {
+      id: repo.uuid,
+      name: repo.name,
+      baseurl: repo.url ? [repo.url] : undefined,
+      gpgkey: repo.gpg_key ? [repo.gpg_key] : undefined,
+      check_gpg: repo.metadata_verification ?? undefined,
+      check_repo_gpg: repo.metadata_verification ?? undefined,
+      module_hotfixes: repo.module_hotfixes ?? undefined,
+      enabled: true,
+    },
+  ];
+}
```

Comment:
There is a `convertSchemaToIBCustomRepo` function in `src/Components/CreateImageWizard/steps/Repositories/components/Utilities.ts` - could we update/reuse that one?

Reply:
That is a very good point. Although they take a different structure as a parameter, they are quite similar. I will try to figure this out. :)

Reply:
Aah, gotcha. Yeah, that would be a lot of work out of scope. Thank you for looking into it! :heart: